SeleniumDemoQA/SeleniumDemoQA/Tests/WebTableTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using SeleniumDemoQA.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumDemoQA.Tests
+{
+    public class WebTableTests: BaseClass
+    {
+        [Test]
+        public void VerifyAddNewRowTable()
+        {
+            var webTablePage = new WebTablePage(_driver);
+            webTablePage.Open();
+
+            int initialRowCount = webTablePage.GetDataRowsCount();
+
+            webTablePage.AddButton();
+            webTablePage.FillFirstNameInput("Elchin");
+            webTablePage.FillLastNameInput("Sangu");
+            webTablePage.FillEmailInput("elchin@ask.com");
+            webTablePage.FillAgeInput("41");
+            webTablePage.FillSalaryInput("50000");
+            webTablePage.FillDepartmentInput("Finance");
+            webTablePage.SubmitButton();
+
+
+            int expectedRowCount = webTablePage.GetDataRowsCount();
```

Comment:
this is not expectedRowCount. Expected count is initialRowCount  + 1
you don't know how many rows will be after adding new row.
I would name it rowsCountAfter

Reply:
got a little confused ;), update...Thanks!